README.md

```diff
@@ -109,11 +109,8 @@ final class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationC
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
 
-        do {
-            let settings = try appSettings.get()
+        if let settings = try appSettings.get() {
```

Comment:
Wouldn't this `try` in get be extraneous? There's a few other `try`s on `gets` in here.